dali/sasha/saxref.cpp

```diff
@@ -2349,7 +2349,10 @@ class CSashaExpiryServer: public ISashaServer, public Thread
             PROGLOG(LOGPFX2 "Deleting %s",lfn);
             try
             {
-                queryDistributedFileDirectory().removeEntry(lfn, UNKNOWN_USER, NULL, INFINITE, true); //MORE:Pass IUserDescriptor
+                /* NB: 0 timeout, meaning fail and skip, if there is any locking contention.
+                 * If the file is locked, it implies it is being accessed.
+                 */
+                queryDistributedFileDirectory().removeEntry(lfn, UNKNOWN_USER, NULL, 0, true); //MORE:Pass IUserDescriptor
```

Comment:
Q: so we just want to try once in doDelete() with the 0 timeout, correct ?

Reply:
Yes, in this case (dealing with expiry). If it's still deemed 'expired' it will try again on the next cycle.